consumer/pdata/timestamp.go

```diff
@@ -12,32 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package internal
+package pdata
```

Comment:
This can be extracted in a separate PR

Reply:
Done in https://github.com/open-telemetry/opentelemetry-collector/pull/1709